content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -59,6 +59,7 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
 | Couchbase                       | `CouchbaseNetClient` 2.2.8+                                                               | `Couchbase`          |
 | Elasticsearch                   | `Elasticsearch.Net` 5.3.0+                                                                | `ElasticsearchNet`   |
 | GraphQL .NET                    | `GraphQL` 2.3.0+                                                                          | `GraphQL`            |
+| gRPC                            | `Grpc.Net.Client`2.3.0+</br>`Grpc.Core` 2.3.0+                                            | `Grpc`               |
```

Comment:
We also instrument Grpc.AspNetCore
```suggestion
| gRPC                            | `Grpc.Net.Client`2.30.0+</br>`Grpc.Core` 2.30.0+</br>`Grpc.AspNetCore` 2.30.0+                                            | `Grpc`               |
```

Reply:
`Grpc.Net.Client`2.3.0+ (.NET Core 3.1 only)</br>`Grpc.Core` 2.3.0+  
Maybe ?

Reply:
Updated in 8b0387a48eaedd222591c2366cb2b47ff6f87c10